core/src/main/scala/org/apache/spark/TestUtils.scala

```diff
@@ -311,6 +312,16 @@ private[spark] object TestUtils {
     current ++ current.filter(_.isDirectory).flatMap(recursiveList)
   }
 
+  /**
+   * Set task resource requirement.
+   */
+  def setTaskResourceRequirement(
+      conf: SparkConf,
+      resourceName: String,
+      resourceCount: Int): SparkConf = {
+    val key = s"$SPARK_TASK_RESOURCE_PREFIX$resourceName$SPARK_RESOURCE_COUNT_SUFFIX"
```

Comment:
`s"${SPARK_TASK_RESOURCE_PREFIX}${resourceName}${SPARK_RESOURCE_COUNT_SUFFIX}"` (easier to read)

Reply:
updated